### ISSUE TITLE
Should "longtitude" field be filtered out?

### ISSUE THREAD

**Issue by n1313** [xkjyeah/singapore-postal-codes#5]
The "longtitude" field is a duplicate of (properly spelled) "longitude" field. Should it be removed from datasets?

**Comment by joelkarunungan** [comment]
From the developer's comments, they are removing their mistake soon (if they haven't already) -- a misspelled field that they had to duplicate to support those who do not misspell it. I removed it from my version.

**Comment by n1313** [comment]
@joelkarunungan yes, I have found the note about deprecating it "in due course", but I am not sure if this means "any time soon". Onemap.sg seems to be updating infrequently, so I don't know how long it'll take them to fix this typo properly.
